NASA-AMMOS/aerie-ui
# Respond to changes in device pixel ratio for canvas components

Closes #682 

- Store and update device pixel ratio on change within Timeline.svelte
- Use the new responsive dpr in Layer* canvas components

TODO:
- [x] Use responsive dpr in LayerGaps

Testing:
1. Open a plan with activities, a line layer, an x-range layer, and a line layer with external resource gaps (see https://github.com/NASA-AMMOS/aerie-ui/pull/607 for directions on how to add a sample profile with gaps)
2. Drag the window to a screen with a higher DPR than the current screen
3. The canvas elements should not appear blurry
